docs/dev/connectors/streamfile_sink.md

```diff
@@ -55,11 +108,18 @@ Basic usage thus looks like this:
 import org.apache.flink.api.common.serialization.SimpleStringEncoder;
 import org.apache.flink.core.fs.Path;
 import org.apache.flink.streaming.api.functions.sink.filesystem.StreamingFileSink;
+import org.apache.flink.streaming.api.functions.sink.filesystem.rollingpolicies.DefaultRollingPolicy;
 
 DataStream<String> input = ...;
 
 final StreamingFileSink<String> sink = StreamingFileSink
-	.forRowFormat(new Path(outputPath), new SimpleStringEncoder<>("UTF-8"))
+    .forRowFormat(new Path(outputPath), new SimpleStringEncoder<>("UTF-8"))
+    .withRollingPolicy(
+        DefaultRollingPolicy.create()
```

Comment:
This can now become `DefaultRollingPolicy.builder()...build()` after your changes ;)

Reply:
👍